src/manager/app/models/ragapp.py

```diff
@@ -35,6 +35,7 @@ def get_default_app_environment(app_name: str) -> Dict[str, str]:
         "MODEL": "gpt-4o-mini",
         "EMBEDDING_MODEL": "text-embedding-3-small",
         "EMBEDDING_DIM": "1024",
+        "CHAT_REQUEST_LIMIT_THRESHOLD": "20",  # 20 chat requests per day
```

Comment:
how about we pass this via manager ENV `CHAT_REQUEST_LIMIT_THRESHOLD`?

Reply:
Ah, great idea. We can then easily change limit in deployment without update the code